scss/_settings.scss

```diff
@@ -95,6 +95,7 @@ $global-menu-nested-margin: 1rem;
 $global-text-direction: ltr;
 $global-flexbox: true;
 $global-prototype-breakpoints: false;
+$global-button-cursor: pointer;
```

Comment:
Please reset the scss changes or do a rebase, they should not be in this PR imo.

Reply:
See 8b81373 : updated app.scss to match v6.4.3

Reply:
> See 8b81373 : updated app.scss to match v6.4.3

Ok, so this is still fine with the PR title? Just asking =)

Reply:
Foundation version on npm is `v6.4.4-rc1`. So we should also have all the changes between `v6.4.3` and `v6.4.4-rc1` here ;)

Reply:
Agreed.

Reply:
Done.